glean_parser/swift.py

```diff
@@ -107,6 +107,18 @@ def variable_name(var: str) -> str:
         return var
 
 
+class Metric:
```

Comment:
Doesn't this contain a "category" of metrics, not a single metric?  Maybe name `Category` or `MetricCollection`?

Reply:
Hm, yeah, actually the naming in below code confused me enough to not be able to come up with a good name here.
I'll swap it to one of your suggestions.